configs/envoyproxy_io_proxy_http3_downstream.yaml

```diff
@@ -49,8 +49,6 @@ static_resources:
             http3_protocol_options:
             http_filters:
             - name: envoy.filters.http.router
-
-  listeners:
```

Comment:
yep @mk46 this was what i meant by removing the second `listener` - just the line - not all of it